.ci_support/linux_64_.yaml

```diff
@@ -5,7 +5,7 @@ boost_cpp:
 c_compiler:
 - gcc
 c_compiler_version:
-- '11'
+- '12'
```

Comment:
The latest rerender in e393be322fba59fd2f85a33831944e56ab94cd7f picked up new gcc and llvm.  If this is problematic, we can pin them back to versions 11 and 14 respectively.